frontend/src/views/Pages/AccountOverview/GddTransactionList.vue

```diff
@@ -11,198 +11,76 @@
+          </template>
 
-          <!-- Collaps Start -->
-          <!-- v-if="
-              (type != 'decay' && decay) ||
-              firstTransaction ||
-              (!firstTransaction && decay === null)
-            " -->
-          <b-collapse class="pb-4" :id="'decay-' + transactionId">
-            <div class="pt-4 pb-4 bg-white border border-muted">
-              <decay-information
-                v-if="decay"
-                decaytyp="new"
-                :balance="balance"
-                :decay="decay"
-                :type="type"
-              />
-              <div v-if="firstTransaction" class="mt-3 mb-3 text-center">
-                <b>{{ $t('decay.first_transaction') }}</b>
-              </div>
+          <template #SEND>
+            <transaction-send
+              class="list-group-item"
+              v-bind="transactions[index]"
+              :decayStartBlock="decayStartBlock"
+            />
+          </template>
 
-              <div
-                v-if="type !== 'decay' && !firstTransaction && decay === null"
-                class="mt-3 mb-3 text-center"
-              >
-                <b>{{ $t('decay.befor_startblock_transaction') }}</b>
-              </div>
-              <div v-if="type === 'decay'" class="mt-3 mb-3">
-                <decay-information
-                  decaytyp="decayLastTransaction"
-                  :gddbalance="gddbalance"
-                  :balance="balance"
-                  :decay="{
-                    balance: balance,
-                    decayStart: parseInt(decayStart),
-                    decayEnd: parseInt(decayEnd),
-                    decayDuration: parseInt(decayDuration),
-                  }"
-                  :type="type"
-                />
-              </div>
-            </div>
-          </b-collapse>
+          <template #RECEIVE>
+            <transaction-receive
+              class="list-group-item"
+              v-bind="transactions[index]"
+              :decayStartBlock="decayStartBlock"
+            />
+          </template>
 
-          <!-- Collaps End -->
-        </div>
-      </div>
-      <pagination-buttons
-        v-if="showPagination"
-        v-model="currentPage"
-        :per-page="pageSize"
-        :total-rows="transactionCount"
-      ></pagination-buttons>
-      <div v-if="transactionCount < 0" class="mt-4 text-center">
-        <span>{{ $t('transaction.nullTransactions') }}</span>
+          <template #CREATION>
+            <transaction-creation
+              class="list-group-item"
+              v-bind="transactions[index]"
+              :decayStartBlock="decayStartBlock"
+            />
+          </template>
+        </transaction-list-item>
       </div>
     </div>
+    <pagination-buttons
+      v-if="showPagination"
+      v-model="currentPage"
+      :per-page="pageSize"
+      :total-rows="transactionCount"
+    ></pagination-buttons>
+    <div v-if="transactionCount < 0" class="mt-4 text-center">
```

Comment:
does this still work?
Why `< 0`?

Reply:
keine ahnung .. ich habe es mal auf <= gesetzt. momentan kann ich leider nichts testen weil 
das problem besteht. 

``` DataTypeNotSupportedError: Data type "Object" in "Transaction.previous" is not supported by "mysql" database.
    at DataTypeNotSupportedError.TypeORMError [as constructor] (/home/tulex/Entwicklung/gradido/database/src/error/TypeORMError.ts:7:9) ```